source/addComment.js

```diff
@@ -1,3 +1,29 @@
-$(document).ready(function(){
-  // add your code here
-})
+$(document).ready(function() {
+    const commentList = $('#comment_list')
+    const commentButton = $('#new_comment_button')
+    const commentForm = $('#new_comment')
+    const commentSubmit = $("input[type='submit']")
+    const newCommentText = $('#new_comment_text')
+    const authorName = $('#authorName')
+
+    const cleanUp = () => {
+        newCommentText.val('')
+        authorName.val('')
+        commentForm.hide();
+    }
+
+    commentButton.click(function() {
```

Comment:
Reach for `on` over `click`: https://stackoverflow.com/questions/9122078/difference-between-onclick-vs-click